content/applications/inventory_and_mrp/manufacturing/management/work_center_time_off.rst

```diff
@@ -0,0 +1,117 @@
+be performed at the other.
+
+.. image:: work_center_time_off/work-center-equipment-tab.png
+   :align: center
+   :alt: The equipment tab on a work center form.
+
+For the work center that will be made unavailable, select the second work center on the
+:guilabel:`Alternative Workcenters` drop-down. Now, Odoo knows to send work orders to the second
+work center when the first is unavailable for any reason.
+
+.. image:: work_center_time_off/alternative-work-center-selection.png
+   :align: center
+   :alt: A work center form configured with an alternative work center.
+
+Add time off for a work center
+------------------------------
+
+With configuration completed, time off can now be assigned to the work center that will be made
+unavailable. Begin by navigating to :menuselection:`Configuration --> Work Centers` and selecting
+the affected work center. Click :guilabel:`Edit` and then the :guilabel:`External link` button next
+to the :guilabel:`Working Hours` drop-down.
+
+.. image:: work_center_time_off/working-hours-button.png
+   :align: center
+   :alt: The Working Hours "External link" button on the work center form.
+
+A pop-up will appear, titled :guilabel:`Open: Working Hours`. The standard working hours for the
+work center will be listed here, along with various other details about it. Since developer mode was
+enabled, there should be a :guilabel:`Time Off` button in the top right of the pop-up. Click it to
+be taken to the :guilabel:`Resource Time Off` page.
+
+.. image:: work_center_time_off/time-off-button.png
+   :align: center
+   :alt: The Time Off button on the Working Hours pop-up.
+
+On this page, click :guilabel:`Create` to configure a new time off entry. On the time off form, note
+the :guilabel:`Reason` for the work center closure (broken, maintenance, etc.), select the affected
+work center as the :guilabel:`Resource`, and choose a :guilabel:`Start Date` and :guilabel:`End
+Date` to specify the period during which the work center will be unavailable. Click :guilabel:`Save`
+and the time off for the work center will be logged in Odoo.
+
+.. image:: work_center_time_off/time-off-form.png
+   :align: center
+   :alt: The "Resource Time Off" form.
+
+Route orders to an alternative work center
+------------------------------------------
+
+Once a work center is within its specified time-off period, work orders sent to it can be
+automatically routed to an alternative work center using the :guilabel:`Plan` button. Begin by
+creating a new MO by selecting :menuselection:`Operations --> Manufacturing Orders --> Create`. On
+the MO form, specify a :guilabel:`Product` that uses the unavailable work center for one if its
+operations. Click :guilabel:`Confirm` to confirm the work order.
+
+On the confirmed work order, select the :guilabel:`Work Orders` tab. By default, the unavailable
+work center will be specified in the :guilabel:`Work Center` column. There should also be a green
```

Comment:
Instead of saying "there should be" or "this should happen", it's a bit stronger and more straightforward to just say "there is".
```suggestion
work center will be specified in the :guilabel:`Work Center` column. There is also a green
```